apps/model_eval_viz/app.py

```diff
@@ -83,58 +84,64 @@ def _get_data(_filepath:str)->gpd.GeoDataFrame:
     title='United States Streamgage Map'
 )
 
-# Instantiate template
-
 # Plotting and Servable execution 
 stream_gage = _get_data(path)
 
 # Widget setup to select multiple states
 state_list = list(states['shapeName'].unique())
 #sort alphabetically
 state_list.sort()
+
+streamgage_input = pn.widgets.TextInput(
+    name='Streamgage Site ID', 
+    placeholder='Streamgage Site ID #',
+    
```

Comment:
Add a description such as this
`description="Enter a column delimited list e.g. 01022500, 01022502"`